roles/openshift_openstack/tasks/check-prerequisites.yml

```diff
@@ -17,6 +17,36 @@
     that: 'shade_result.rc == 0'
     msg: "Python module shade is not installed"
 
+- include_tasks: prerequisites/provider-network-check.yml
```

Comment:
would be nice to include a kuryr one too :)

Reply:
Haha, sure - I can group the existing kuryr checks together and make sure that the other kuryr parameters are filled out - does that sound right?